__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data")
+
+beforeEach(() => { return seed (data) });
+afterAll(() => { return db.end() });
+
+describe("/api/topics", () => {
+    it("GET - 200, returns an array of all topics", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+            body.topics.forEach(topic => {
+                expect(typeof topic.slug).toBe("string")
+                expect(typeof topic.description).toBe("string")
+            })
+        })
+    })
+    it("GET - 404, responds with an error when given an invalid endpoint", () => {
+        return request(app)
+        .get("/api/toopicss")
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("Not Found")
```

Comment:
This is a small thing 😅 but in the test above you destructured the body from the response (line 15), and here you haven't. Staying consistent in your styling makes your code look as professional and clear as possible ✨